zerver/tests/test_upload.py

```diff
@@ -2058,6 +2101,23 @@ def test_get_emoji_url(self) -> None:
         expected_url = f"https://{bucket}.s3.amazonaws.com/{path}"
         self.assertEqual(expected_url, url)
 
+        emoji_name = "animated_image.gif"
+
+        path = RealmEmoji.PATH_ID_TEMPLATE.format(realm_id=realm_id, emoji_file_name=emoji_name)
+        still_path = RealmEmoji.STILL_PATH_ID_TEMPLATE.format(
+            realm_id=realm_id, emoji_file_name_without_ext=emoji_name.replace(".gif", "")
+        )
+
+        url = zerver.lib.upload.upload_backend.get_emoji_url("animated_image.gif", realm_id)
+        still_url = zerver.lib.upload.upload_backend.get_emoji_url(
+            "animated_image.gif", realm_id, still=True
+        )
+
+        expected_url = f"https://{bucket}.s3.amazonaws.com/{path}"
+        self.assertEqual(expected_url, url)
+        expected_still_url = f"https://{bucket}.s3.amazonaws.com/{still_path}"
+        self.assertEqual(expected_still_url, still_url)
```

Comment:
We should make sure we assert the value of `still_url` for a non-animated image somewhere in this test file.